app/ios/iosinterface.mm

```diff
@@ -80,6 +89,53 @@ static QImage fromUIImage( UIImage *image )
   return result;
 }
 
+
+static NSObject *readExifAttribute( NSString *imagePath, NSString *tag )
+{
+  NSData *data = [NSData dataWithContentsOfFile:imagePath];
+  CGImageSourceRef source = CGImageSourceCreateWithData( ( __bridge CFDataRef )data, NULL );
+  NSDictionary *metadata = [( NSDictionary * )CGImageSourceCopyPropertiesAtIndex( source, 0, NULL )autorelease];
+
+  // Modify EXIF tag for GPS group
+  // TODO more generic handle of EXIF (tagskCGImageProperty*)
+  NSString *newTag = [tag stringByReplacingOccurrencesOfString: @"GPS" withString:@""];
```

Comment:
should be done in IosUtils::getExitInfo

Reply:
Id rather have separate obj-c utils function and pure cpp functions - `readExifAttribute`
returns `NSString` and its dealing with other `NS` classes. 
 Also `fromUIImage` could go to obj-c utils beside `readExifAttribute`.
What do you think?